packages/wallet/src/Wallets/BaseWallet.ts

```diff
@@ -551,7 +558,23 @@ export class BaseWallet implements ObservableWallet {
       utxo: this.utxo
     });
 
-    this.getPubDRepKey().catch(() => void 0);
+    const getPubDRepKey = async (): Promise<Ed25519PublicKeyHex | undefined> => {
+      if (isBip32PublicCredentialsManager(this.#publicCredentialsManager)) {
+        return (await this.#publicCredentialsManager.bip32Account.derivePublicKey(util.DREP_KEY_DERIVATION_PATH)).hex();
+      }
+
+      return undefined;
+    };
+
+    this.governance = {
+      getPubDRepKey,
+      isRegisteredAsDRep$: createDRepRegistrationTracker({
+        historyTransactions$: this.transactions.history$,
+        pubDRepKeyHash$: from(getPubDRepKey().then(getDRepKeyHash))
+      })
+    };
+
+    this.governance.getPubDRepKey().catch(() => void 0);
```

Comment:
I'm not sure why this was needed as it doesn't appear to have any side effects, but I believe it's obsolete because it is being called just above `pubDRepKeyHash$: from(getPubDRepKey().then(getDRepKeyHash))`
```suggestion
```

Reply:
done in [63c5f161c3dd48eef2a5e3b0225d41f662530e99](https://github.com/input-output-hk/cardano-js-sdk/pull/1192/commits/63c5f161c3dd48eef2a5e3b0225d41f662530e99)